lib/modules/datasource/github-releases/index.ts

```diff
@@ -239,22 +208,26 @@ export class GithubReleasesDatasource extends Datasource {
    *  - Sanitize the versions if desired (e.g. strip out leading 'v')
    *  - Return a dependency object containing sourceUrl string and releases array
    */
+  // istanbul ignore next
```

Comment:
why this?

Reply:
This PR is intended to be temporary so we can test in the app

Reply:
yes, but there are tests for those functions, so coverage should be good. 🤔